Linux/sgx/buildenv.mk

```diff
@@ -73,11 +73,13 @@ endif
 ifeq ($(DEBUG), 1)
 	OBJDIR := debug
 	OPENSSL_LIB := libsgx_tsgxssl_cryptod.a
+	OPENSSL_SSL_LIB := libsgx_tsgxssl_ssld.a
 	TRUSTED_LIB := libsgx_tsgxssld.a
 	UNTRUSTED_LIB := libsgx_usgxssld.a
 else
 	OBJDIR := release
 	OPENSSL_LIB := libsgx_tsgxssl_crypto.a
+	OPENSSL_SSL_LIB := libsgx_tsgxssl_ssl.a
```

Comment:
After a successful build, the generated static library will be in  different  folders by the configurations.
package/lib64/
├── cve_2020_0551_cf
│   ├── libsgx_tsgxssl.a
│   └── libsgx_tsgxssl_crypto.a
├── cve_2020_0551_load
│   ├── libsgx_tsgxssl.a
│   └── libsgx_tsgxssl_crypto.a
├── libsgx_tsgxssl.a
├── libsgx_tsgxssl_crypto.a
└── libsgx_usgxssl.a
I only found libsgx_tsgxssl_ssl.a under package/lib64/ for non-debug build. Please make change in the sgx/Makefile.